packages/postgres-database/src/simcore_postgres_database/migration/versions/ff33dcf20b44_adding_column_product_name_to_groups_.py

```diff
@@ -0,0 +1,55 @@
+"""adding column product_name to groups_extra_properties
+
+Revision ID: ff33dcf20b44
+Revises: e84904303e2b
+Create Date: 2023-02-15 14:06:52.562223+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "ff33dcf20b44"
+down_revision = "e84904303e2b"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+
+    conn = op.get_bind()
+    default_product_name = conn.scalar(
+        "SELECT name from products ORDER BY priority LIMIT 1"
+    )
+
+    op.add_column(
+        "groups_extra_properties",
+        sa.Column(
+            "product_name",
+            sa.VARCHAR(),
+            nullable=False,
+            server_default=default_product_name,
```

Comment:
Q: did you test it on AWS-prod database already? You can easily import the database. I can show you if needed.

Reply:
The config from AWS-prod cannot be pulled. I've got the one from master, since it is pretty similar to the one in AWS-prod (regarding products).

All worked well.

Reply:
Thanks for the input